app/scripts/views/customizer/CustomizerAppView.js

```diff
@@ -22,7 +22,8 @@ define(['backbone', 'CustomizerViewerView', 'CustomizerHeaderView', 'CustomizerW
 
     render: _.once(function() {
 
-      (new CustomizerViewer()).render();
+      new ThreeViewer({app: this.model});
```

Comment:
Again, let's not pass the app into `ThreeViewer.`  This mistake is analogous to the one I made when injecting `Workspace` into `Node`, and @marimano we'll tell you how fun that was to resolve.  :(